mmdet/core/evaluation/text_evaluation.py

```diff
@@ -52,7 +52,8 @@ def masks_to_rects(masks, is_rle):
 def polygon_from_points(points):
     """ Returns a Polygon object to use with the Polygon2 class from a list of 8 points:
         x1,y1,x2,y2,x3,y3,x4,y4 """
-
+    if len(points) == 1:
```

Comment:
In the doc string above it's mentioned that `points` should be "a list of 8 points". In what case `len(points) == 1`?

Reply:
This fix is needed to prevent the model from falling if more than 8 points come to the input. It can happen if the word was annotated with polygon instead of bounding box. But we decided that using standard datasets and conversion scripts, the user will not face this problem, so that makes this check unnecessary.  I'll remove it.